packages/ckeditor5-dev-env/tests/release-tools/utils/transform-commit/transform-commit-utils.js

```diff
@@ -41,46 +41,170 @@ describe( 'dev-env/release-tools/utils/transform-commit', () => {
 				stubs.getPackageJson.returns( {
 					name: 'test-package'
 				} );
 
-				expect( () => transformCommit.linkGithubIssues( '' ) )
+				expect( () => transformCommit.linkToGithubIssue( '' ) )
 					.to.throw( Error, 'The package.json for "test-package" must contain the "bugs" property.' );
 			} );
 
-			it( 'marks references to issues as links to GitHub issues #1', () => {
+			it( 'replaces "#ID" with a link to GitHub issue (packageJson.bugs as a string)', () => {
 				stubs.getPackageJson.returns( {
 					name: 'test-package',
 					bugs: '/issues'
 				} );
 
-				expect( transformCommit.linkGithubIssues( 'Some issue #1.' ) )
+				expect( transformCommit.linkToGithubIssue( 'Some issue #1.' ) )
 					.to.equal( 'Some issue [#1](/issues/1).' );
 			} );
 
-			it( 'marks references to issues as links to GitHub issues #2', () => {
+			it( 'replaces "#ID" with a link to GitHub issue (packageJson.bugs as an object)', () => {
 				stubs.getPackageJson.returns( {
 					name: 'test-package',
 					bugs: {
-						url: '/issues'
+						url: 'https://github.com/ckeditor/ckeditor5-dev/issues'
 					}
 				} );
 
-				expect( transformCommit.linkGithubIssues( 'Some issue #1.' ) )
-					.to.equal( 'Some issue [#1](/issues/1).' );
+				expect( transformCommit.linkToGithubIssue( 'Some issue #1.' ) )
+					.to.equal( 'Some issue [#1](https://github.com/ckeditor/ckeditor5-dev/issues/1).' );
+			} );
+
+			it( 'does not replace if the hash belongs to other repository', () => {
+				stubs.getPackageJson.returns( {
+					name: 'test-package',
+					bugs: {
+						url: 'https://github.com/ckeditor/ckeditor5-dev/issues'
+					}
+				} );
+
+				expect( transformCommit.linkToGithubIssue( 'organization/repository#1' ) )
+					.to.equal( 'organization/repository#1' );
 			} );
 		} );
 
+		describe( 'linkToGithubRepository()', () => {
+			it( 'makes a link to GitHub if a comment matches to "organization/repository"', () => {
+				expect( transformCommit.linkToGithubRepository( 'ckeditor/ckeditor5-dev' ) )
+					.to.equal( '[ckeditor/ckeditor5-dev](https://github.com/ckeditor/ckeditor5-dev)' );
+			} );
+
+			it( 'makes a link to GitHub issue if a comment matches to "organization/repository#ID"', () => {
+				expect( transformCommit.linkToGithubRepository( 'ckeditor/ckeditor5-dev#2' ) )
+					.to.equal( '[ckeditor/ckeditor5-dev#2](https://github.com/ckeditor/ckeditor5-dev/issues/2)' );
+			} );
+
+			it( 'does not make a link from a comment which is a scoped package', () => {
+				expect( transformCommit.linkToGithubRepository( '@ckeditor/ckeditor5-dev' ) )
+					.to.equal( '@ckeditor/ckeditor5-dev' );
+			} );
+
+			it( 'does not make a link from a comment which is a path', () => {
+				expect( transformCommit.linkToGithubRepository( 'i/am/a/path' ) )
+					.to.equal( 'i/am/a/path' );
+			} );
+
+			it( 'does not make a link if a comment does not match to "@organization/repository"', () => {
```

Comment:
`organization/repository`

Reply:
In that case, I would enable the link.

Reply:
I would like to create a link only if the string matches in 100%, so `org/repo` is ok but `org/repo/` isn't ok. Maybe other people will help us with the decision (@Reinmar, I'm looking at you :D)